syntax/yang.vim

```diff
@@ -85,8 +85,8 @@ hi def link yangConditional Conditional
 hi def link yangType Type
 
 " arbitrary mappings
-hi def link yangKeyword Type
-hi def link yangStatement Type
+hi def link yangKeyword Keyword
+hi def link yangStatement Statement
 hi def link yangModule Type
 hi def link yangDateArg Conditional
 hi def link yangLengthArg Conditional
```

Comment:
`Conditional` seems to get highlighted the same as `Statement` on my color scheme (solarized-light/dark).  With your change, this makes, for example, `revision 2014-05-08` all come out in one color.  What do you think about changing `yangDateArg` and `yangLengthArg` to `Number`?  Seems a little closer to their semantic purpose...